nat-lab/tests/telio.py

```diff
@@ -399,10 +401,18 @@ async def on_stdout(stdout: str) -> None:
         self._runtime = Runtime()
         self._events = Events(self._runtime)
         self._router = new_router(self._connection)
-        self._process = self._connection.create_process(
-            [tcli_path, "--less-spam", f"-f {self._telio_features.to_json()}"]  # type: ignore
-        )
-
+        if telio_v3:
```

Comment:
what if I were to run this with `WINDOWS_VM` or `MAC_VM`

Reply:
mark doesn't solve the issue, since it doesn't prevent people from using `telio_v3` in other tests on other connections besides linux. Either add binary to windows/mac VMs and make it accessible through `get_libtelio_binary_path` or atleast raise and Exception or something to inform users that `telio_v3` can only be used with linux

Reply:
Added a note above the test to inform about the test limitation to run on different platforms.